docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -30,6 +30,9 @@ To address those mentioned issues and make the management of `HttpClient` instan
 - `HttpClient` already has the concept of delegating handlers that could be linked together for outgoing HTTP requests. You register HTTP clients into the factory and you can use a Polly handler to use Polly policies for Retry, CircuitBreakers, and so on.
 - Manage the lifetime of `HttpClientMessageHandlers` to avoid the mentioned problems/issues that can occur when managing `HttpClient` lifetimes yourself.
 
+> [!NOTE]
+> `HttpClientFactory` is tightly tied to the Microsoft.Extensions.DependencyInjection implementation. You might want to read this [GitHub discussion]((https://github.com/aspnet/Extensions/issues/1345)) about using other dependency injection containers.
```

Comment:
```suggestion
> `HttpClientFactory` is tightly tied to the `Microsoft.Extensions.DependencyInjection` implementation. For more information about using other dependency injection containers, see this [GitHub discussion](https://github.com/aspnet/Extensions/issues/1345).
```

Reply:
clarify what Microsoft.Extensions.DependencyInjection is perhaps? It's a NuGet package, right? And how is it tied to that? Is it part of that package or what?

Reply:
Yep, at this level it's more conceptual than package specific, will clarify it.